index.html

```diff
@@ -302,7 +302,7 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="ok"><a href="https://golang.org/pkg/crypto/tls/#Config">yes</a></td>
             <td class="ok"><a href="https://golang.org/pkg/crypto/tls/#Config">yes</a></td>
             <td class="ok"><a href="https://golang.org/doc/go1.6#http2">yes</a></td>
-            <td class="alert">no</td>
+            <td class="ok"><a href="https://github.com/golang/go/issues/9671#issuecomment-437431164">yes</a></td>
```

Comment:
And this can link to https://golang.org/doc/go1.12#tls_1_3 -- or, if you prefer actual package docs, https://golang.org/pkg/crypto/tls/